handlers/main.yml

```diff
@@ -7,10 +7,11 @@
   command: postmap {{ postfix_default_database_type }}:{{ postfix_virtual_aliases_file }}
 
 - name: postmap sasl_passwd
-  command: postmap {{ postfix_default_database_type }}:{{ postfix_sasl_passwd_file }}
+  command: postmap -p {{ postfix_default_database_type }}:{{ postfix_sasl_passwd_file }}
```

Comment:
What does this do?

Reply:
this makes sure that the permissions on the db file are correct